test/models/order_item_test.rb

```diff
@@ -1,9 +1,27 @@
 require "test_helper"
+require "pry"
 
 describe OrderItem do
-  let(:order_item) { OrderItem.new }
+  let(:order_item) { order_items(:one) }
 
   it "must be valid" do
-    value(order_item).must_be :valid?
+    expect(order_item.valid?).must_equal true
+  end
+
+  it "must require a quantity" do
```

Comment:
this is where my comment from the order_item.rb is relevant. (tldr: i dont think we can have invalid items in our yml files. we need to make valid items invalid IN our tests)